tests/pipeline/snapshots/snap_test_pipeline.py

```diff
@@ -6,290 +6,9 @@
 
 snapshots = Snapshot()
 
-snapshots["TestPipeline.test_atm_fraud_example test-pipeline"] = {
```

Comment:
snapshot wasn't up to date, this has been moved to https://github.com/bakdata/kpops/blob/5d4c9d293232f8db8daf6d591abbc2310172c0df/tests/pipeline/snapshots/snap_test_example.py